### PR TITLE
fix(stepfunctions): Support 'parameters' in TaskProps and PassProps

### LINKED ISSUE #3453: PassProp Should accept parameters just like StateProp.
**Note: for support questions, please first reference our [documentation](https://docs.aws.amazon.com/cdk/api/latest), then use [Stackoverflow](https://stackoverflow.com/questions/ask?tags=aws-cdk)**. This repository's issues are intended for feature requests and bug reports.

* **I'm submitting a ...**
  - [ ] :beetle: bug report

* **What is the current behavior?**
*If the current behavior is a :beetle:bug:beetle:: Please provide the steps to reproduce*

Pass State Props currently does not support adding parameters  : https://github.com/aws/aws-cdk/blob/97b1168db536f013fa0db50257608d2413d61293/packages/%40aws-cdk/aws-stepfunctions/lib/states/pass.ts#L53

* **What is the expected behavior (or behavior of feature suggested)?**

It should accept parameters just like State.

https://github.com/aws/aws-cdk/blob/97b1168db536f013fa0db50257608d2413d61293/packages/%40aws-cdk/aws-stepfunctions/lib/states/state.ts#L35

* **What is the motivation / use case for changing the behavior or adding this feature?**

parameters gives more freedom to mainipulate json input.

https://docs.aws.amazon.com/step-functions/latest/dg/input-output-inputpath-params.html

* **Please tell us about your environment:**

  - CDK CLI Version: xx.xx.xx
  - Module Version: xx.xx.xx
  - OS: [all | Windows 10 | OSX Mojave | Ubuntu | etc... ]
  - Language: [all | TypeScript | Java | Python ]

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. associated pull-request, stackoverflow, gitter, etc)

### PR DESCRIPTION
**Pull request by alukach**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

I think this should address #3453, however it's such a simple change that I'm worried that I may be missing something larger.  I am not entirely familiar with CDK's inner-workings.

fixes #3453

### CODE REVIEW

File: packages/@aws-cdk/aws-stepfunctions/lib/states/pass.ts [hunk 1]
```diff
@@ -96,6 +96,16 @@ export interface PassProps {
...
+
+    /**
+     * Parameters pass a collection of key-value pairs, either static values or JSONPath expressions that select from the input.
+     *
+     * @see
+     * https://docs.aws.amazon.com/step-functions/latest/dg/input-output-inputpath-params.html#input-output-parameters
+     *
+     * @default No parameters
+     */
+    readonly parameters?: { [name: string]: any };
```
Comment (rix0rrr) [inline packages/@aws-cdk/aws-stepfunctions/lib/states/pass.ts]: This allows passing of the field. Does it get rendered? Please add a test to prove that it does.

Comment (jogold) [reply packages/@aws-cdk/aws-stepfunctions/lib/states/pass.ts]: I'm also facing an issue with the missing `paramaters` props. Wouldn't it be better to have `PassProps` and `TaskProps` extend `StateProps`?

**Comment by alukach** [comment]
Sorry everyone, I did a `rebase` on my branch and apparently it automatically set all of you as a requested reviewer.  I have no idea how that happened, but given the branch conflicts I'm going to go ahead and close this PR and create a new branch and PR without all of this drama.

File: packages/@aws-cdk/aws-stepfunctions/lib/states/pass.ts (hunk 1 above)
Comment (alukach) [reply packages/@aws-cdk/aws-stepfunctions/lib/states/pass.ts]: @jogold I agree that `PassProps` and `TaskProps` have a lot of overlap with `StateProps` and could benefit from a refactor, however my goal here is getting `parameters` supported ASAP. As such, I'm only making changes to work within the current codebase design.

**Pull request closed without merging**